src/components/Cards.js

```diff
@@ -26,7 +39,24 @@ export default class Card {
       });
   }
 
-  _handleTrashButton() {
+  setIsLiked(isLiked) {
+    this._isLiked = isLiked;
+    this.setButtonState();
+  }
+
+  setButtonState() {
+    if (this._isLiked) {
+      this._cardElement
+        .querySelector(".card__like-button")
+        .classList.add("card__like-button_active");
```

Comment:
You've already found `this._likeButton`  in line 22. Please, use it everywhere

`Class fields` (the so-called **class variables** with `this`) are accessible inside any class method. You shouldn’t pass **class fields** into the call of any method or search them again because you can use class fields inside all methods.